README.md

```diff
@@ -19,24 +19,41 @@ either with skaffold (Kubernetes), with docker-compose, or manually.
 
 This is the mode used by default. Two files are currently available:
 - `values.yaml` (default): for networks started with Kubernetes
-- `values-docker-compose.yaml`: for networks started with docker-compose, or started manually
+- `local-values.yaml`: for networks started with docker-compose, or started manually
 
 To run the tests using the default `values.yaml` file:
 
+Make sure you have pulled the different substra-tools images:
+```
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.0.1
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.1.0
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.2.0
+```
+
+With skaffold
+
 ```
 make test
 ```
 
-To run the tests using the provided `values-docker-compose.yaml` (or a custom config file):
+To run the tests using the provided `local-values.yaml` (or a custom config file):
+
+With docker-compose or locally
 
 ```
-SUBSTRAT_CONFIG_FILEPATH=values-docker-compose.yaml make test
+SUBSTRAT_CONFIG_FILEPATH=local-values.yaml make test
 ```
 
 ## Using a skaffold file
 
 It is possible to use a `substra-backend` skaffold file as the source for the network configuration:
-
 ```
 SUBSTRAT_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Install substra cli
```

Comment:
Could be moved also to the prerequisites section?

Reply:
done